src/module.c

```diff
@@ -870,6 +870,7 @@ int64_t commandKeySpecsFlagsFromString(const char *s) {
         else if (!strcasecmp(t,"delete")) flags |= CMD_KEY_DELETE;
         else if (!strcasecmp(t,"channel")) flags |= CMD_KEY_CHANNEL;
         else if (!strcasecmp(t,"incomplete")) flags |= CMD_KEY_INCOMPLETE;
+        else if (!strcasecmp(t,"variable_flags")) flags |= CMD_KEY_VARIABLE_FLAGS;
```

Comment:
need to document that flag.
but come to think of it, is it really needed for modules?
we don't yet have a way to pass flags in RedisModule_KeyAtPos.
i suggest removing it form this PR, and add it in another PR (one that extends RedisModule_KeyAtPos).
thing is that adding a module api in this PR would make it a major decision.

Reply:
These lines will cause merge conflicts to #9944. I prefer that this PR is not touching these lines.

Reply:
created an issue https://github.com/redis/redis/issues/10189
@zuiderkwast @guybe7 FYI (somewhat related to https://github.com/redis/redis/pull/10108)